src/hooks/usePersistentAspectRatio.ts

```diff
@@ -36,8 +36,9 @@ export default function usePersistentAspectRatio(url: string): Result {
     if (state === State.init && nonSvgUrl) {
       const lowResUrl = getLowResUrl(nonSvgUrl) as string;
       setState(State.loading);
-      Image.getSize(
+      Image.getSizeWithHeaders(
         lowResUrl,
+        { cache: 'reload' },
```

Comment:
we already cache the result and not ask if we have this cached so this layer of cache is redundant

Reply:
maybe this made a difference?